<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test_checkout_tests.TestDeliverProduct-20240424224337" tests="7" file="test_checkout_tests.py" time="0.027" timestamp="2024-04-24T22:43:37" failures="0" errors="7" skipped="0">
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_already_checkout" time="0.012" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_checkin_code_not_found_date_signature" time="0.003" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_checkout_ok" time="0.002" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_guest_checkout_bad_date_signature" time="0.002" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_guest_checkout_no_date" time="0.002" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_guest_checkout_no_store_check_in" time="0.002" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
	<testcase classname="test_checkout_tests.TestDeliverProduct" name="test_guest_checkout_store_check_in_is_empty" time="0.002" timestamp="2024-04-24T22:43:37">
		<error type="AttributeError" message="'HotelManager' object has no attribute 'regex_match'"><![CDATA[Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 587, in run
    self._callSetUp()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 546, in _callSetUp
    self.setUp()
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/venv/lib/python3.10/site-packages/freezegun/api.py", line 827, in wrapper
    result = func(*args, **kwargs)
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/unittest/python/test_checkout_tests.py", line 38, in setUp
    localizer = hotel_mngr.room_reservation(credit_card="5105105105105100",
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 66, in room_reservation
    arrival_date, credit_card, num_days, phone_number, room_type = self.validate_data_room_res(
  File "/Users/ellaz-j/PycharmProjects/G88.2024.T06.EG3/src/main/python/uc3m_travel/hotel_manager.py", line 97, in validate_data_room_res
    self.regex_match(id_card, r)
AttributeError: 'HotelManager' object has no attribute 'regex_match'
]]></error>
	</testcase>
</testsuite>
